src/Servers/IIS/IIS/test/Common.FunctionalTests/RequestResponseTests.cs

```diff
@@ -156,12 +156,10 @@ public async Task PassesThroughCompressionInProcess()
     }
 
     [ConditionalFact]
-    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/26294")]
-    [SkipNonHelix("This test takes 5 minutes to run")]
```

Comment:
Running successfully 💯% of the time isn't sufficient reason to un-quarantine. Does the following apply (from [the doc](https://github.com/dotnet/aspnetcore-internal/blob/main/docs/operations/unquarantine_tests.md))❔

> Find the set of tests that were supposedly fixed at least 30 days ago.